src/handlers/tests/interventions-processor-lambda.test.ts

```diff
@@ -405,5 +464,89 @@ describe('intervention processor handler', () => {
       expect(logAndPublishMetric).toHaveBeenCalledWith('CONFIDENCE_LEVEL_TOO_LOW');
       expect(logger.warn).toHaveBeenCalledWith('Received interventions has low level of confidence: P1');
     });
+
+    it('It should log an error and continue if the inner message body is not valid JSON', async () => {
```

Comment:
absolute nit pick, apols:
```suggestion
    it('should log an error and continue, if the inner message body is not valid JSON', async () => {
```